src/lib/subname-helpers.ts

```diff
@@ -99,7 +99,7 @@ export function decodeDNSPacketBytes(buf: Uint8Array): [string | null, string |
   let hex = Buffer.from(buf).toString("hex");
   let firstLabel = "";
 
-  if (len === 0) {
+  if (!len) {
```

Comment:
FYI @shrugs, this change fixes this failing test case:
![image](https://github.com/user-attachments/assets/486248ea-521f-44f8-a487-c856f3f73d65)

Reply:
I've skipped the test so we could stick to the original `decodeDNSPacketBytes` implementation.